ui/package.json

```diff
@@ -143,14 +144,14 @@
     "yargs-parser": "^13.1.1"
   },
   "optionalDependencies": {
-    "@babel/core": "^7.5.5",
-    "@storybook/addon-knobs": "^5.1.10",
-    "@storybook/addon-links": "^5.1.10",
-    "@storybook/addon-notes": "^5.1.10",
-    "@storybook/addon-viewport": "^5.1.10",
-    "@storybook/addons": "^5.1.10",
-    "@storybook/ember": "^5.1.10",
-    "babel-loader": "^8.0.6",
+    "@babel/core": "^7.10.4",
```

Comment:
i updated the rest of our babel dependencies since we were starting to fall behind anyway.